modules/basis/src/basis-loader.js

```diff
@@ -0,0 +1,8 @@
+const EXTENSIONS = ['basis'];
+
+export default {
+  name: 'basis',
+  extensions: EXTENSIONS,
+  parse: x => console.log('parse', x),
+  loadAndParse: x => console.log('loadAndParse', x)
```

Comment:
You don't need loadAndParse. Just `parse` and maybe `parseSync`

Reply:
ok